src/main/java/gomedic/commons/core/Messages.java

```diff
@@ -17,11 +17,10 @@
 
     public static final String MESSAGE_UNKNOWN_COMMAND = "Sorry, %s is an invalid command.";
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
-    public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person index provided is invalid";
     public static final String MESSAGE_INVALID_ACTIVITY_ID = "The activity id doesn't exist in the list";
     public static final String MESSAGE_INVALID_DOCTOR_ID = "The doctor id doesn't exist in the list";
     public static final String MESSAGE_INVALID_PATIENT_ID = "The patient id doesn't exist in the list";
-    public static final String MESSAGE_PERSONS_LISTED_OVERVIEW = "%1$d persons listed!";
+    public static final String MESSAGE_DOCTORS_LISTED_OVERVIEW = "%1$d doctors listed!";
```

Comment:
What is this command for ? as in I thought we have `list t/doctor` already ?

Reply:
This line is for the `find` command. So after running the `find` command, it will show "5 doctors listed" if 5 of them pass through the filter. 

This was from the original AB3's implementation of `find`, just that I changed it to "doctors" from "persons" for now. I think i will just change it to "%1$d items listed!" instead, so that we can all share the same message.